opba-protocols/protocol-bpmn-shared/src/main/java/de/adorsys/opba/protocol/bpmnshared/config/flowable/FlowableProperties.java

```diff
@@ -1,28 +1,55 @@
 package de.adorsys.opba.protocol.bpmnshared.config.flowable;
 
 import lombok.Data;
+import org.jetbrains.annotations.NotNull;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
+import java.time.Duration;
 import java.util.List;
 
 import static de.adorsys.opba.protocol.bpmnshared.config.flowable.ConfigConst.FLOWABLE_SHARED_CONFIG_PREFIX;
 
-/**
- * Configures which classes can be serialized/deserialized in JSON form and configures threshold when to use
- * {@link JsonCustomSerializer} or {@link LargeJsonCustomSerializer}
- */
+
 @Data
 @Configuration
-@ConfigurationProperties(prefix = FLOWABLE_SHARED_CONFIG_PREFIX + "serialization")
+@ConfigurationProperties(prefix = FLOWABLE_SHARED_CONFIG_PREFIX)
 public class FlowableProperties {
+    /**
+     * Configures number of retries for flowable engine and for retry template config
+     */
+    @SuppressWarnings("checkstyle:MagicNumber")
+    private int numberOfRetries = 3;
 
-    private List<String> serializeOnlyPackages;
+    @NotNull
+    private Serialization serialization;
 
-    @SuppressWarnings("checkstyle:MagicNumber")
-    private int maxLength = 2048;
+    @NotNull
```

Comment:
It is the wrong annotation, you should use NotNull from `javax.validation.constraints.NotNull`. Same on other fields